new_dawn_server/settings.py

```diff
@@ -33,7 +33,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['.elasticbeanstalk.com', '127.0.0.1', 'localhost']
+ALLOWED_HOSTS = ['.elasticbeanstalk.com', '127.0.0.1', 'localhost', "192.168.1.18"]
```

Comment:
What is this host?

Reply:
That is my local IP. I used it to test simulator and physical phone together.

Reply:
By default our iOS HttpUtil is pointing to “localhost” when running non-prod mode.  Maybe you can change that to a deterministic IP such as 127.0.0.1. I don’t think it is ideal to include your actual IP address here.

Reply:
I agree with you that I should not put my own ip address. But in order to allow phone or any other device to be able to access that django local server, it needs to use a same IP address as my wifi. So, anything connects my wifi can get on that server, without making the server public.

Reply:
I can change it to 0.0.0.0

Reply:
Yes. For IOS side, I do also need to change it to my IP.

Reply:
Got it. Let’s change it to 0.0.0.0 then for now.

Reply:
Ok!